_content/articles/wat-phum-thmei-palm-leaf-library_menchaca-philip.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Cambodia’s Wat Phum Thmei Palm Leaf Library and the Resilience of Buddhist Texts"
+authors:
+  - "Philip Menchaca"
+source_url: "https://tricycle.org/article/wat-phum-thmei-texts/#:~:text=Importantly%2C%20the%20Wat%20Phum%20Thmei,complex%20cultural%20and%20spiritual%20landscape."
```

Comment:
Oh? The article is behind a paywall for you? I'm able to access it... 🤔 

Anyway: please trim off the #:~:text= suffix that Google adds